compiler/src/dotty/tools/dotc/typer/Namer.scala

```diff
@@ -877,16 +877,16 @@ class Namer { typer: Typer =>
     protected def addAnnotations(sym: Symbol): Unit = original match {
       case original: untpd.MemberDef =>
         lazy val annotCtx = annotContext(original, sym)
-        original.setMods: 
+        original.setMods:
```

Comment:
Not sure if it's worth including this diff here as it's only formatting. It might make future "git blaming" harder.

Reply:
Unfortunately, it is automatic. I think that vscode removes trailing whitespaces by default, so I would have to specifically open it in a different editor and add the whitespaces manually every time. I don't think it's worth it unless you feel strongly about it.

Reply:
I don't feel strongly about it 🙂